controllers/cluster_controller_test.go

```diff
@@ -211,4 +219,173 @@ var _ = Describe("Cluster Reconciler", func() {
 			return instance.Finalizers
 		}, timeout).Should(BeEmpty())
 	})
+
+	It("Should successfully set Status.ControlPlaneInitialized on the cluster object if controlplane is ready", func() {
+		cluster := &clusterv1.Cluster{
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: "test-",
+				Namespace:    "default",
```

Comment:
```suggestion
				Namespace:    v1.NamespaceDefault,
```

(since you used this below)